core/src/main/java/tc/oc/pgm/PGMPlugin.java

```diff
@@ -40,23 +40,7 @@
 import tc.oc.pgm.api.player.MatchPlayer;
 import tc.oc.pgm.api.setting.SettingKey;
 import tc.oc.pgm.api.setting.SettingValue;
-import tc.oc.pgm.commands.AdminCommands;
-import tc.oc.pgm.commands.ClassCommands;
-import tc.oc.pgm.commands.CycleCommands;
-import tc.oc.pgm.commands.DestroyableCommands;
-import tc.oc.pgm.commands.FreeForAllCommands;
-import tc.oc.pgm.commands.GoalCommands;
-import tc.oc.pgm.commands.InventoryCommands;
-import tc.oc.pgm.commands.JoinCommands;
-import tc.oc.pgm.commands.MapCommands;
-import tc.oc.pgm.commands.MapPoolCommands;
-import tc.oc.pgm.commands.MatchCommands;
-import tc.oc.pgm.commands.ModeCommands;
-import tc.oc.pgm.commands.ObserverCommands;
-import tc.oc.pgm.commands.SettingCommands;
-import tc.oc.pgm.commands.StartCommands;
-import tc.oc.pgm.commands.TeamCommands;
-import tc.oc.pgm.commands.TimeLimitCommands;
+import tc.oc.pgm.commands.*;
```

Comment:
Why this change?

Reply:
It was either my IDE or Maven formatting, any issues with doing this?

Reply:
It's just necessary :P

Reply:
It looks nicer? I can undo it

Reply:
Do i really have to? I dont see the point of filling up the import list when you are going to register all commands and match modules anyways :p

Reply:
You don't have to. I'm always trying keep local code standards and change only what's necessary. :P

Reply:
You should avoid wildcard imports, they're bad